apache/incubator-zeppelin
# Make zeppelin work with CDH5.7.0

### What is this PR for?
The downloadable zeppelin install wasn;t working with CDH 5.7.0. I had to change a few things to make it work nicely together.

### What type of PR is it?
Bug Fix

### Todos

### What is the Jira issue?
https://www.mail-archive.com/users@zeppelin.incubator.apache.org/msg03471.html

### How should this be tested?
Install local cdh5.7.0 (see https://github.com/krisgeus/ansible_local_cdh_hadoop)
build zeppeling (with this patch) with the following options:
mvn clean package -DskipTests -Pspark-1.6 -Phadoop-2.6 -Dspark.version=1.6.0-cdh5.7.0 -Dhadoop.version=2.6.0-cdh5.7.0 -Pvendor-repo

### Screenshots (if appropriate)

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? Didn't verify
* Does this needs documentation? Don't think so
